airbyte-webapp/src/components/ui/InfoBox/InfoBox.tsx

```diff
@@ -21,7 +21,7 @@ export const InfoBox: React.FC<React.PropsWithChildren<Props>> = ({ children, cl
   return (
     <Box className={className}>
       {icon && <FontAwesomeIcon size="lg" icon={icon} />}
-      <div>{children}</div>
```

Comment:
Shouldn't this be the responsibility of the children? 
I remove it in order to be able to have full control on the children node

Reply:
I guess the only reason might have been vertical alignment issues why this was here. But if it works in all places without it, no reason to keep it around.